_posts/2020-07-30-deploy-create-react-app-aws-pulumi.md

```diff
@@ -0,0 +1,351 @@
+---
+layout: post
+title:  "Deploying a Create React App to AWS with pulumi"
+date:   2020-77-39 00:00:00 +0100
+categories: pulumi aws react
+---
+
+# Getting started with create react app
+
+So you have just created your first app in create react app.
+
+https://reactjs.org/docs/create-a-new-react-app.html
+
+You built it, change some source files. It works! Well, it works on your machine. 
+What's next? How do you actually get it out there, running? 
+You can build it but you want to ship it
+
+The aim of this guide is to provide a step-by-step, incremental improvement journey. 
+What's the point in showing you how to build the castle, if it's not clear why one needs anything more than a shack?
+
+This is not just a cookbook recipie that assumes things will work out. 
+When possible, debuig and diagnostic commands are run to make sure things are in the state that they need to be. 
+
+This will be a big journey but if we can break it up into smaller substeps. 
```

Comment:
I think maybe you meant:
`This will be a big journey but if we can break it up into smaller iterative substeps, we'll go from a shack to a castle.`